### DISCUSSION TITLE
Error framer-motion in Next JS

### DISCUSSION THREAD

**Discussion by padi-dev-quocnd** [chakra-ui/chakra-ui#6926] [Q&A]
Help me ! i cant resolve this issue when use framer-motion in NextJS

**Comment by padi-dev-quocnd** [comment]
![screenshot_1667228357](https://user-images.githubusercontent.com/113343485/199039474-8461a71c-6722-4eab-b31b-9d2f2e691566.png)

**Reply by simonpeters** [reply]
maybe make an issue?
